logging/bin/es_nodeport_disable_open.sh

```diff
@@ -12,5 +12,11 @@ export SVC=v4m-es-client-service
 
 log_info "Removing NodePort for Elasticsearch instance [$SVC] in [$LOG_NS]"
 
-kubectl -n "$LOG_NS" patch svc "$SVC" --type='json' -p '[{"op":"replace","path":"/spec/type","value":"ClusterIP"},{"op":"remove","path":"/spec/ports/0/nodePort"},{"op":"remove","path":"/spec/ports/1/nodePort"},{"op":"remove","path":"/spec/ports/2/nodePort"}]' 
+if [[ $KUBE_SERVER_VER =~ v1.18 ]]; then
+   kubectl -n "$LOG_NS" patch svc "$SVC" --type='json' -p '[{"op":"replace","path":"/spec/type","value":"ClusterIP"},{"op":"remove","path":"/spec/ports/0/nodePort"},{"op":"remove","path":"/spec/ports/1/nodePort"},{"op":"remove","path":"/spec/ports/2/nodePort"}]' 
+elif [[ $KUBE_SERVER_VER =~ v1.19 ]]; then
+   kubectl -n "$LOG_NS" patch svc "$SVC" --type='json' -p '[{"op":"replace","path":"/spec/type","value":"ClusterIP"},{"op":"remove","path":"/spec/ports/0/nodePort"},{"op":"remove","path":"/spec/ports/1/nodePort"},{"op":"remove","path":"/spec/ports/2/nodePort"},{"op":"remove","path":"/spec/ports/3/nodePort"}]' 
+else
+   kubectl -n "$LOG_NS" patch svc "$SVC" --type='json' -p '[{"op":"replace","path":"/spec/type","value":"ClusterIP"}]'
+fi
 
```

Comment:
@BryanEllington  Can you take a look at this please.  K8s 1.19.x adds an additional port to be removed, so we were failing on some clusters. 😞  And K8s 1.20 added the following (which makes things much easier/better):
> Certain fields on Service objects will be automatically cleared when changing the service's type to a mode that does not need those fields. For example, changing from type=LoadBalancer to type=ClusterIP will clear the NodePort assignments, rather than forcing the user to clear them. (#95196, @thockin) [SIG API Machinery, Apps, Network and Testing]
--from [Kubernetes 1.20 release notes](https://v1-20.docs.kubernetes.io/docs/setup/release/notes/)

Reply:
It's nice that 1.20+ automatically handles this annoying issue, but we still need to support 1.19. The code looks correct to me.